aw_qt/trayicon.py

```diff
@@ -159,12 +159,11 @@ def run(manager, testing=False):
     if sys.platform == "darwin":
         from Foundation import NSUserDefaults
         style = NSUserDefaults.standardUserDefaults().stringForKey_('AppleInterfaceStyle')
-        if style == "Dark":
-            icon = QIcon(":/white-monochrome-logo.png")
-        else:
-            icon = QIcon(":/black-monochrome-logo.png")
+        icon = QIcon(":/black-monochrome-logo.png")
     else:
         icon = QIcon(":/logo.png")
+    # Allow tray programs to use filters for changing the icon's color
+    icon.setIsMask(True)
```

Comment:
I think this should only be set for the black monochrome logo on macOS.

Reply:
Oh of course. I was thinking it would also enable better dark mode for other platforms but should probably test before enabling it everywhere. I'll change that.

Reply:
Pushed fix.